### PR TITLE
Update documentation sorting

### PR DESCRIPTION
**Pull request by TOsmanov**
### Motivation and context

This PR contain:
- Update documentation sorting
- Add images for context image section
- Add "getting started" page and add create_a_new_task.gif
- Add images for context image section
- Fix incorrect links on *.md files in the repository

documentation structure:

    │   faq.md
    │   getting_started.md
    │
    ├───administration
    │   │
    │   ├───advanced
    │   │       analytics.md
    │   │       backup_guide.md
    │   │       installation_automatic_annotation.md
    │   │       mounting_cloud_storages.md
    │   │
    │   └───basics
    │           AWS-Deployment-Guide.md
    │           installation.md
    │           REST_API_guide.md
    │
    ├───developing
    │   │   format.md
    │   │
    │   └───contributing
    │           branching-model.md
    │           bug-reports.md
    │           coding-style.md
    │           development-environment.md
    │           feature-requests.md
    │           pull-requests.md
    │           setup-additional-components.md
    │           using-the-issue-tracker.md
    │
    └───manual
        │
        ├───advanced
        │   │   ai-tools.md
        │   │   analytics.md
        │   │   annotation-with-polylines.md
        │   │   annotation-with-rectangle-by-4-points.md
        │   │   annotation-with-tags.md
        │   │   attribute-annotation-mode-advanced.md
        │   │   automatic-annotation.md
        │   │   cli.md
        │   │   context-images.md
        │   │   dataset_manifest.md
        │   │   data_on_fly.md
        │   │   downloading-annotations.md
        │   │   filter.md
        │   │   models.md
        │   │   opencv-tools.md
        │   │   projects.md
        │   │   review.md
        │   │   search.md
        │   │   shape-grouping.md
        │   │   shape-mode-advanced.md
        │   │   shortcuts.md
        │   │   task-synchronization.md
        │   │   track-mode-advanced.md
        │   │   xml_format.md
        │   │
        │   ├───annotation-with-cuboids
        │   │       creating-the-cuboid.md
        │   │       editing-the-cuboid.md
        │   │
        │   ├───annotation-with-points
        │   │       liner-interpolation-with-one-point.md
        │   │       points-in-shape-mode.md
        │   │
        │   ├───annotation-with-polygons
        │   │       automatic-borders.md
        │   │       creating-mask.md
        │   │       edit-polygon.md
        │   │       manual-drawing.md
        │   │       track-mode-with-polygons.md
        │   │
        │   └───formats
        │           format-camvid.md
        │           format-coco.md
        │           format-cvat.md
        │           format-datumaro.md
        │           format-icdar.md
        │           format-imagenet.md
        │           format-labelme.md
        │           format-market1501.md
        │           format-mot.md
        │           format-mots.md
        │           format-tfrecord.md
        │           format-vggface2.md
        │           format-voc.md
        │           format-widerface.md
        │           format-yolo.md
        │
        └───basics
                attribute-annotation-mode-basics.md
                authorization.md
                basic-navigation.md
                controls-sidebar.md
                creating_an_annotation_task.md
                interface.md
                objects-sidebar.md
                settings.md
                shape-mode-basics.md
                task-details.md
                top-panel.md
                track-mode-basics.md
                types-of-shapes.md
                vocabulary.md
                workspace.md

### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [x] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [x] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2021 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

**Comment by azhavoro** [comment]
https://azhavoro.github.io/cvat/docs/

**Comment by azhavoro** [comment]
@TOsmanov Hi,
My comments:
  - [ ] I think  it will be better to rename the `Formats` section under `Developing` to `How to add a new annotation format support`
  - [ ] `Developing` - > `Developing and contributing` What do you think? Also please remove subsection `Contributing` as discussed earlier.

**Comment by TOsmanov** [comment]
> @TOsmanov Hi,
> My comments:
> 
> * I think  it will be better to rename the `Formats` section under `Developing` to `How to add a new annotation format support`
> * `Developing` - > `Developing and contributing` What do you think? Also please remove subsection `Contributing` as discussed earlier.

@azhavoro Hi,
Do I need to divide the "Developing and contributing" section into "basics" and "advanced" as it is done with other sections?

**Comment by azhavoro** [comment]
> @azhavoro Hi,
> Do I need to divide the "Developing and contributing" section into "basics" and "advanced" as it is done with other sections?

I think you don't

**Comment by azhavoro** [comment]
Updated version of the documentation is here: https://azhavoro.github.io/cvat/docs/

File: site/content/en/docs/_index.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ---
 title: 'Documentation'
 linkTitle: 'Documentation'
+description: 'Welcome to the documentation of Computer Vision Annotation Tools!'
```
Comment (nmanovic) [inline site/content/en/docs/_index.md:4]:
```suggestion
description: 'Welcome to the documentation of Computer Vision Annotation Tool.'
```

File: site/content/en/docs/manual/basics/creating_an_annotation_task.md [hunk 2]
```diff
@@ -35,19 +35,19 @@ weight: 2
      The following actions are available here:
 
      1. Set the attribute’s name.
-     1. Choose the way to display the attribute:
+     2. Choose the way to display the attribute:
```
Comment (nmanovic) [inline site/content/en/docs/manual/basics/creating_an_annotation_task.md:38]: Probably it should be `1.` in accordance with markdown specification.

**Pull request merged**